tools/depends/target/mariadb/Makefile

```diff
@@ -16,8 +24,8 @@ $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	rm -rf $(PLATFORM); mkdir -p $(PLATFORM)/build
 	cd $(PLATFORM); $(ARCHIVE_TOOL) $(ARCHIVE_TOOL_FLAGS) $(TARBALLS_LOCATION)/$(ARCHIVE)
 	cd $(PLATFORM); patch -p1 -i ../01-android.patch
-	sed -ie 's| "DYNAMIC" | "STATIC" |' $(PLATFORM)/cmake/plugins.cmake
-	cd $(PLATFORM)/build; $(CMAKE) -DAUTH_GSSAPI=OFF -DWITH_UNITTEST:BOOL=OFF -DWITH_EXTERNAL_ZLIB:BOOL=ON -DWITH_CURL:BOOL=OFF ..
+	cd $(PLATFORM); patch -p1 -i ../02-symbol_rename-gnutls_clash.patch
```

Comment:
Rename the function by using a preprocessor defines. Otherwise you always have to recheck the patch file.

Reply:
Hope this is what you meant with the updated patch. Seemed to work and be cleaner locally. If not, if you wouldnt mind elaborating a little.